### PR TITLE
Add list priority command

### LINKED ISSUE #118: List Priority COmmand

### PR DESCRIPTION
**Pull request by Mamtha3005**
@CS2103AUG2016-W15-C3/code-quality 
@CS2103AUG2016-W15-C3/documentation 
@CS2103AUG2016-W15-C3/testing 

Please review. 

This closes #118

### CODE REVIEW

**Comment by turtle96** [comment]
Testing done.

File: src/test/java/seedu/taskell/testutil/TypicalTestTasks.java [hunk 1]
```diff
@@ -14,21 +14,21 @@
 
     public TypicalTestTasks() {
         try {
-            archivePastEmails =  new TaskBuilder().withDescription("Archieve past emails").withTaskType(Task.EVENT_TASK).withTaskPriority("0")
+            archivePastEmails =  new TaskBuilder().withDescription("Archieve past emails").withTaskType(Task.EVENT_TASK).withTaskPriority("1")
```
Comment (marcusngwenjian) [inline src/test/java/seedu/taskell/testutil/TypicalTestTasks.java]: Magic values. Use constants in TaskPriority class

File: src/main/java/seedu/taskell/model/ModelManager.java [hunk 2]
```diff
@@ -100,25 +105,34 @@ public void updateFilteredListToShowAll() {
     }
 
     @Override
-    public void updateFilteredTaskList(Set<String> keywords){
+    public void updateFilteredTaskList(Set<String> keywords) {
         updateFilteredTaskList(new PredicateExpression(new NameQualifier(keywords)));
     }
-    
+
+    // @@ author A0142073R
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/model/ModelManager.java]: remove whitespace between @@ and author

File: src/main/java/seedu/taskell/logic/parser/Parser.java [hunk 3]
```diff
@@ -136,6 +139,21 @@ public Command parseCommand(String userInput) {
...
 
+    private Command prepareListPriority(String args) {
+        if (args.isEmpty()) {
+            return new IncorrectCommand(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ListPriorityCommand.MESSAGE_USAGE));
+        }
+        StringTokenizer st = new StringTokenizer(args.trim(), " ");
+        String intValue = st.nextToken();
+        int targetIdx = Integer.valueOf(intValue);
+        if (targetIdx < 0 || targetIdx > 3) {
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/logic/parser/Parser.java:159]: Magic number. Please make it as a constant in task priority class

File: src/test/java/guitests/ListPriorityCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import static org.junit.Assert.assertTrue;
+
+public class ListPriorityCommandTest extends TaskManagerGuiTest {
+
+    @Test
+    public void listPriority() throws IllegalValueException {
+
+        // list 2 tasks in the list
+        TestTask[] currentList = td.getTypicalTasks();
+        String priority = "1";
```
Comment (marcusngwenjian) [inline src/test/java/guitests/ListPriorityCommandTest.java]: Magic string. Please use constant in task priority

**Comment by marcusngwenjian** [comment]
Code Quality reviewed.
Please resolve issue #133

**Comment by Mamtha3005** [comment]
This closes #133

**Pull request closed without merging**